pkg/config/node.go

```diff
@@ -694,3 +713,80 @@ func PopulateNodeAddresses(kubeconfigPath string, node *Node) {
 		}
 	}
 }
+
+func getConfigWithCloudLBIPs(kubeconfigPath, clusterConfigPath, resolvConfPath string, clusterLBConfig ClusterLBConfig) (node Node, err error) {
+	var apiLBIP, apiIntLBIP, ingressIP net.IP
+	nodes := []Node{}
+	ipCount := 0
+	if len(clusterLBConfig.ApiIntLBIPs) > len(clusterLBConfig.IngressLBIPs) {
+		ipCount = len(clusterLBConfig.ApiIntLBIPs)
+	} else {
+		ipCount = len(clusterLBConfig.IngressLBIPs)
+	}
+	for i := 0; i < ipCount; i++ {
+		if i < len(clusterLBConfig.ApiIntLBIPs) {
+			apiIntLBIP = clusterLBConfig.ApiIntLBIPs[i]
+		} else {
+			apiIntLBIP = nil
```

Comment:
Because in L718 you wrote `var apiIntLBIP net.ip` you do not need to assign it to `nil` explicitly as it's the value you get from start

Reply:
apiIntLBIP and other LB IP values are being set within a loop starting at L726 so its value could have been !nil in the previous iteration.

Reply:
Yeah, I see it now... In that case this deserves at least a comment in Simple English directly in the code or some kind of a simplification...

As I read this whole loop now (L726-L730) is that when you finish the loop, you want `apiIntLBIP` to have a value of the last element of the `clusterLBConfig.ApiIntLBIPs` slice. Is that correct? If yes, in pseudocode that would be

```
apiIntLBIP = get_last_element(clusterLBConfig.ApiIntLBIPs)
```

Do I get the desired behaviour right? If yes, I'd rather do something like

```
apiIntLBIP = clusterLBConfig.ApiIntLBIPs[len(clusterLBConfig.ApiIntLBIPs)-1]
```

instead of iteration in which every time we assign a value only to ignore it afterwards and use the last assignment.

If my reasoning is wrong and your desired behaviour is different then ignore what I wrote, but that means a comment is more than desired

Reply:
I was trying to follow the precedence set by https://github.com/openshift/baremetal-runtimecfg/blob/master/pkg/config/node.go#L421-L447. There are definitely other ways to implement this too. 
Adding a comment for clarity.